domainservice/build.gradle.kts

```diff
@@ -0,0 +1,36 @@
+plugins {
+    java
+    `java-test-fixtures`
```

Comment:
```suggestion
```

seems we don't need these two plugins because they are already applied via the spring-conventions